CONTRIBUTING.md

```diff
@@ -74,8 +74,8 @@ git push origin <YourBranchName>
 
 - fix: a commit of the type fix patches a bug in your codebase (this correlates with PATCH in semantic versioning).
 - feat: a commit of the type feat introduces a new feature to the codebase (this correlates with MINOR in semantic versioning).
-- BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in semantic           versioning). A BREAKING CHANGE can be part of commits of any type.
-- types other than fix: and feat: are allowed, for example @commitlint/config-conventional (based on the the Angular convention) recommends build:, chore:, ci:, docs:, style:,     refactor:, perf:, test:, and others.
+- BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in semantic        versioning). A BREAKING CHANGE can be part of commits of any type.
+- types other than fix: and feat: are allowed, for example @commitlint/config-conventional (based on the the Angular convention) recommends build:, chore:, ci:, docs:, style:,   refactor:, perf:, test:, and others.
```

Comment:
```suggestion
+ types other than fix: and feat: are allowed, for example @commitlint/config-conventional (based on the the Angular convention) recommends build:, chore:, ci:, docs:, style:, refactor:, perf:, test:, and others.
```

Reply:
@muthuannamalai12 these were marked as resolved with no comment or change?

Reply:
Sorry. It was my mistake I thought it was resolved. Now I have made the required changes